portfolio/src/main/webapp/index.html

```diff
@@ -4,26 +4,33 @@
 <link rel="stylesheet" href="style.css">
 <script src="script.js"></script>
 
-<nav class="navbar">
-    <div class="nav-left-side">
-        <a href="/index.html" class="nav-button navbar-item">NR</a>
-        <a href="/works.html" class="nav-button navbar-item">Work</a>
-        <a href="/contact.html" class="nav-button navbar-item">Contact</a>
+<body onload="loadName()">
+	<nav class="navbar">
+		<div class="nav-left-side">
+			<a href="/index.html" class="nav-button navbar-item">NR</a>
+			<a href="/works.html" class="nav-button navbar-item">Work</a>
+			<a href="/contact.html" class="nav-button navbar-item">Contact</a>
+		</div>
+		<div class="nav-right-side">
+			<a href="https://www.linkedin.com/in/roland-naijuka" target="_blank"
+				class="nav-button navbar-item"><img src="/images/linkedin.png" alt="My LinkedIn"></a>
+				<a href="mailto:naijuka.roland@outlook.com" target="_blank"
+					class="nav-button navbar-item"><img src="/images/mail.png" alt="Email me"></a>
+		</div>
+	</nav>
+	<div class="content">
+		<div class="content-body">
+			<p class="tooltip" id="welcome-note">
```

Comment:
UX nit: I love the tooltip! But it was hard for me to find at first haha
Totally optional suggestion, what about putting the tooltip over your image, and adding a "hover over me!" text under the picture or something like that, so that users know there's an interesting tooltip there as well?

Reply:
> UX nit: I love the tooltip! But it was hard for me to find at first haha
> Totally optional suggestion, what about putting the tooltip over your image, and adding a "hover over me!" text under the picture or something like that, so that users know there's an interesting tooltip there as well?

Sorry about that. I will definitely put it on the picture and let users know about it